packages/tools/e2e-tests/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "./node_modules/@kadena-dev/heft-rig/tsconfig-base.json",
+  "compilerOptions": {
+    "types": ["node"],
+    "rootDir": "./",
```

Comment:
Why we wouldn't have `src` as `rootDir`?

Reply:
It wasn't before. Good Catch.